CHANGELOG.md

```diff
@@ -2,6 +2,53 @@
 
 Notable changes to QMCPACK will be documented in this file. 
 
+## [Unreleased]
+
+### Notes
+
+This section describes improvements made to the the development branch
+since the last release.
+
+### QMCPACK updates
+
+* Improved population control during DMC equilibration. Reduces variance on larger runs.
+* All particle move VMC and DMC algorithms enabled, tests added.
+* Reptation Monte Carlo (RMC) enabled, tests added.
+* Significantly improved AFQMC implementation.
+* Added NiO based VMC and DMC performance tests and description in manual. Wavefunction files accessed via QMC_DATA.
+* Added DMC tests with locality and t-moves aapproximations.
+* Added AFQMC tests.
+* Added test of real space QMC restart capabilities.
+* Added tests for several estimators.
+* Added unit test for DMC walker propagation, effective core potentials, OhmmsPETE.
+* To avoid filesystem limitations, QMC_SYMLINK_TEST_FILES can be set to symlink (1) or copy test files (0).
+* Status of t-moves now printed.
```

Comment:
I think this was reverted by me. I'm still waiting for Ray to resubmit a better way printing.